modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -302,6 +302,104 @@ TEST_P(GAPI_Streaming, SmokeTest_VideoConstSource_NoHang)
     EXPECT_EQ(ref_frames, test_frames);
 }
 
+TEST_P(GAPI_Streaming, SmokeTest_AutoMeta)
+{
+    cv::GMat in;
+    cv::GMat in2;
+    cv::GMat roi = cv::gapi::crop(in2, cv::Rect{1,1,256,256});
+    cv::GMat blr = cv::gapi::blur(roi, cv::Size(3,3));
+    cv::GMat out = blr - in;
+
+    auto testc = cv::GComputation(cv::GIn(in, in2), cv::GOut(out))
+        .compileStreaming(cv::compile_args(cv::gapi::use_only{GetParam()}));
+
+    cv::Mat in_const = cv::Mat::eye(cv::Size(256,256), CV_8UC3);
+    cv::Mat tmp;
```

Comment:
If I understand correctly, the purpose of this test is to check if the AutoMeta feature works. Is it necessary to use the constant source (here and in the following test as well)? This issue has been checked already in the previous test, as I can see..

Reply:
In the  previous test `SmokeTest_VideoConstSource_NoHang` we set the metadata in `compileStreaming()` and after run pipeline. In that test `SmokeTest_AutoMeta_VideoArra` we check feature when metadata comes only in `setSource()`

Reply:
OK